test/write.jl

```diff
@@ -1,4 +1,6 @@
 using CSV, Dates, WeakRefStrings, CategoricalArrays, Tables
+using FilePathsBase
```

Comment:
Why is this needed?

Reply:
Possibly you only need the line after: `using FilePathsBase: /`. Without that the FilePaths test fail.

Reply:
@oxinabox @rofinn , do either of you know why this would now be needed? It seems odd to me. Did FilePathsBase stop exporting something or something?

Reply:
Yes, `/` and `join` no longer extend the base methods as requested by some core dev team members. If you want those methods to work with a path type then you need to do `using FilePathsBase: /, join`. FWIW, we might be able to think of a way for FilePathsBase to be a test-only dependency if we loosen a couple of type restrictions?